client/flutter/ios/Runner/AppDelegate.swift

```diff
@@ -21,7 +23,7 @@ import Flutter
     }
 
     if #available(iOS 10.0, *) {
-      UNUserNotificationCenter.current().delegate = self as? UNUserNotificationCenterDelegate
+      UNUserNotificationCenter.current().delegate = self
```

Comment:
Can we change this back to the default? I know it's unnecessary but it's good practice to keep only the changes that fix this particular issue to this PR and do other refactoring in subsequent PRs.
```suggestion
      UNUserNotificationCenter.current().delegate = self as? UNUserNotificationCenterDelegate
```